app/templates/base.html

```diff
@@ -73,8 +73,7 @@
           </div>
           <div class="spacer d-print-none my-0"></div>
         </div>
-      </div>
-
+        </div>
```

Comment:
You added the wrong indentation and I don't think you needed to make any changes to this file